common/djangoapps/student/tests/test_tasks.py

```diff
@@ -0,0 +1,44 @@
+"""
+Tests for the Sending activation email celery tasks
+"""
+
+import mock
+
+from django.test import TestCase
+from django.conf import settings
+from student.tasks import send_activation_email
+
+
+class SendActivationEmailTestCase(TestCase):
+    """
+    Test for send activation email to user
+    """
+    @mock.patch('time.sleep', mock.Mock(return_value=None))
+    @mock.patch('student.tasks.log')
+    @mock.patch('django.contrib.auth.models.User')
+    def test_send_email(self, mock_user, mock_log):
+        """
+        Test Send activation email exception
```

Comment:
Can we describe what we actually testing here? i.e.  we are testing retries logic works as defined.